test/quantization/test_quantize_script.py

```diff
@@ -1400,11 +1400,47 @@ def forward(self, x):
                 return self.bn(x)
 
         data = [(torch.rand((1, 3, 10, 10), dtype=torch.float), torch.randint(0, 1, (1,), dtype=torch.long)) for _ in range(2)]
-        model = self._test_op_impl(M, data, "quantized::batch_norm2d")
+        model = self._test_op_impl(M(), data, "quantized::batch_norm2d")
 
         FileCheck().check_not("aten::batch_norm") \
                    .run(model.graph)
 
+    def test_qbatch_norm_relu(self):
+        class BNRelu(torch.nn.Module):
+            def __init__(self, inplace):
+                super(BNRelu, self).__init__()
+                self.bn = torch.nn.BatchNorm2d(3).to(torch.float)
+                self.relu = torch.nn.ReLU(inplace=inplace)
+
+            def forward(self, x):
+                return self.relu(self.bn(x))
+
+        # Note Fusion for functional Relu with inplace argument isn't currently supported in fusion patterns.
+        class BNFuncRelu(torch.nn.Module):
+            def __init__(self, inplace):
+                super(BNFuncRelu, self).__init__()
+                self.bn = torch.nn.BatchNorm2d(3).to(torch.float)
+
+            def forward(self, x):
+                return F.relu(self.bn(x), False)
+
+        class BNFuncInplaceRelu(torch.nn.Module):
+            def __init__(self, inplace):
+                super(BNFuncInplaceRelu, self).__init__()
+                self.bn = torch.nn.BatchNorm2d(3).to(torch.float)
+
+            def forward(self, x):
+                return F.relu(self.bn(x), True)
+
+        data = [(torch.rand((1, 3, 10, 10), dtype=torch.float), torch.randint(0, 1, (1,), dtype=torch.long)) for _ in range(2)]
+        for Model in [BNRelu, BNFuncRelu, BNFuncInplaceRelu]:
+            for inplace in [True, False]:
+                model = self._test_op_impl(Model(inplace), data, "quantized::batch_norm2d_relu")
```

Comment:
please change this loop as well, we don't need to loop through [True, False] anymore

Reply:
We still need to test BNRelu model for different inplace argument.

Reply:
Good catch! Updated it